### ISSUE TITLE
Did you get any far on this?

### ISSUE THREAD

**Issue by ezjones** [s33plusplus/Migraine-Forecast#1]
I am trying to do something similar. I will add an arduino later to the mix.
https://ezjones.github.io/migranalyzer/

**Comment by s33plusplus** [comment]
Yeah, I did get the Arduino to serve up barometric pressure samples over the serial port, with a hacky python script to poll, timestamp, and log the samples. I'll commit the changes I've made as soon as I can, but it really needs a rewrite to handle exceptions gracefully without leaving the serial port stuck "in-use".

I haven't quite got to writing the UI or prediction yet either, so at the moment it's just a datalogger. I'll add the arduino sketch source along with the parts I'm using to the repo, it should be easy enough to work with.

**Comment by ezjones** [comment]
Cool. Are you planning to leave it capturing data? I am capturing the weather report, headache amount and will add an arduino to the mix.

**Comment by ezjones** [comment]
It would be cool to capture from different cities

**Comment by s33plusplus** [comment]
Yup, it's been capturing for probably 6 months now. The arduino sketch and sensor setup is so dead simple that pretty much anybody who wants to could replicate the circuit on a breadboard.

The code currently pairs the datapoints with a ctime format timestamp, so I could match it up with weather reports and such. I could add support for logging it to some web-based sensor data aggregator too, but I'm not sure if they'd be cool with the freqency I'm sampling at (every 10s currently).

I've also been considering finding a more efficent way to log migraines themselves, like a smartphone app or something. I'm really bad at remembering to write that stuff down by hand.

Thanks for reminding me about this project by the way! It kind of got put on the backburner when I injured by wrist late last year, and I sort of forgot about it. Feel free to message me if you have any ideas or want to collaberate some time!
